tests/data/test-data.sql

```diff
@@ -198,6 +199,9 @@ insert into map_version (id, description, max_players, width, height, version, f
   (16, 'SCMP 015', 8, 512, 512, 2, 'maps/scmp_015.v0002.zip', 0, 15),
   (17, 'SCMP 015', 8, 512, 512, 3, 'maps/scmp_015.v0003.zip', 0, 15);
 
+insert into map_version (id, description, max_players, width, height, version, filename, hidden, ranked, map_id) values
+  (18, 'Sneaky_Map', 8, 512, 512, 1, "maps/neroxis_map_generator_sneaky_map.zip", 0, 0, 16);
+
```

Comment:
Might be more straightforward to add that map to the previous insert statement (and set `ranked` of all maps that are there already).

Reply:
Done